canonical-web-and-design/vanilla-framework
# Update social icon library

## Done

- Updated all social icons to use the official brand assets
- Removed `$color-social-icon-background` and `$color-social-icon-foreground` global variables
- Removed icon hover states

## QA

- Pull code
- Run `./run serve --watch`
- http://0.0.0.0:8101/examples/patterns/icons/icons-social/ or https://vanilla-framework-canonical-web-and-design-pr-2641.run.demo.haus/examples/patterns/icons/icons-social/
- You should see a new social icon and inside a demo example of docs, as we can't serve this push due to docs being on a previous release of our framework (will revert once happy with QA)

## Details

Fixes https://github.com/canonical-web-and-design/design-vanilla-framework/issues/436

## Screenshots

<img width="1406" alt="Screenshot 2019-11-13 at 15 07 47" src="https://user-images.githubusercontent.com/17748020/68776073-64a13880-0627-11ea-9812-f4b53d2fdd23.png">
